masterlist.yaml

```
@@ -6721,6 +6727,9 @@ plugins:
     msg:
       - <<: *patch3rdParty_LotD
         condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_ArsMetallica_Patch.esp")'
+    tag:
+      - -Invent.Remove
```

It's an Invent tag in the plugin header.
Maybe remove that and replace it with Invent.Add.
```suggestion
      - -Invent
      - Invent.Add
```

---

That's a better solution, thanks!

---

It seems `-Invent` doesn't apply `-Invent.Remove` or `-Invent.Add`.
So the tags will still be `Invent.Add` & `Invent.Remove`.
We could ignore the `Invent` tag and treat it as only having `Invent.Add` & `Invent.Remove`.
But this tag is in the plugin header, and if it's removed, it would end up with no tags.
So we have to treat it as having the tags, and potentially not having the tags.
Suggesting removal of `Invent` doesn't really matter, as it won't have any effect in bash.

```yaml
    tag:
      - Invent.Add
      - -Invent.Remove
```

---

That's because WB doesn't internally recognize `Invent` as a tag anymore. As soon as it sees any deprecated tags in a description etc., it replaces them with their updated versions - in this case both `Invent.Add` and `Invent.Remove`. So if the masterlist then tries to remove the `Invent` tag, that does nothing, because no `Invent` tag is applied to that plugin anymore (only the two Add/Remove tags).

---

Yeah, I just assumed at the time that the removal of deprecated tags was taken into account when updating.